draft-yu-ccamp-optical-resource-pm-yang.md

```diff
@@ -1,7 +1,7 @@
 ---
 coding: utf-8
 
-title: A YANG Data Model for Optical Resource Performance Monitoring
+title: A YANG Data Model for Device-level Resource Performance Monitoring
```

Comment:
I think making the title neutral and stricting the resource type within limited range (network element, interface, board, termination point, tunnel termination point) is not suitable. In addition, generalizing the resource types (such as VPN, tunnel) would be controversial( the model will confict with existing VPN PM model and tunnel PM model). 

So I suggest we keep the title as device-level, keep the resource types as currently defined, and clarify the model's applicability in the main body of the document (As far as I know, the model we defined is only used for NBI in IP domain, not for SBI usage.  If both SBI and NBI are used in optical domain, then the applicability of the model needs more discuccsion ) .

Reply:
Ok for me to limit the applicability of this model only at the NBI (it seems I have misunderstood this point from the call)

I think the term 'device-level' is very confusing since it will imply in many people SBI rather than NBI applicability

I am also not sure the term 'device-level' is resolving the potential conflict with existing VPN PM model and tunnel PM model

Please also note that some PM data is already defined for packet topology model in [draft-ietf-teas-yang-l3-te-topo](https://datatracker.ietf.org/doc/html/draft-ietf-teas-yang-l3-te-topo)

IMHO, the key difference here is that we are defining a generic mechanism to retrieve data using the traditional 15min/24h current data and history data definitions. In G.7710 this is called PM data collection

What about "A YANG Data Model for Resource Performance Monitoring Data Collection"?